mmv1/products/compute/InterconnectAttachmentGroup.yaml

```diff
@@ -0,0 +1,258 @@
+          - 'PRODUCTION_NON_CRITICAL'
+          - 'PRODUCTION_CRITICAL'
+          - 'NO_SLA'
+          - 'AVAILABILITY_SLA_UNSPECIFIED'
+    required: true
+  - name: 'logicalStructure'
+    type: NestedObject
+    description: |
+      An analysis of the logical layout of Attachments in this
+      group. Every Attachment in the group is shown once in this structure.
+    properties:
+      - name: 'regions'
+        type: Array
+        description: |
+          The regions Attachments in this group are in.
+        item_type:
+          type: NestedObject
+          properties:
+            - name: 'region'
+            - name: 'metros'
+              type: Array
+              description: |
+                The metros of Attachments in this group in this region.
+              item_type:
+                type: NestedObject
+                properties:
+                  - name: 'metro'
+                    type: String
+                    description: |
+                      The name of the metro, as a three-letter lowercase
+                      string like "iad". This is the first component of the location of an
+                      Interconnect.
+                  - name: 'facilities'
+                    type: Array
+                    description: |
+                      The facilities used for this group's Attachments'
+                      Interconnects.
+                    item_type:
+                      type: NestedObject
+                      properties:
+                        - name: 'facility'
+                          type: String
+                          description: |
+                            The name of a facility, like "iad-1234".
+                        - name: 'zones'
+                          type: Array
+                          description: |
+                            The zones that Interconnects in this facility are
+                            present in.
+                          item_type:
+                            type: NestedObject
+                            properties:
+                              - name: 'zone'
+                                type: String
+                                description: |
+                                  The zones that Attachments in this group are present
+                                  in, in the given facilities.  This is inherited from their
+                                  Interconnects.
+                              - name: 'attachment'
+                                type: Array
+                                description: |
+                                  URLs of Attachments in the given zone, to the given
+                                  region, on Interconnects in the given facility and metro. Every
+                                  Attachment in the AG has such an entry.
+                                item_type:
+                                  type: String
+    output: true
+  - name: 'configured'
```

Comment:
Same here on `output: true` - if the entire object is output only each field within also needs it (that said, the [docs](https://cloud.google.com/compute/docs/reference/rest/v1/interconnectAttachmentGroups/insert) don't indicate that `effectiveSla` or `blockerType` are output only, but they look to me like they might be)